setup.py

```diff
@@ -133,9 +133,7 @@ def read(*names, **kwargs):
 
 try:
     from retriever.compile import compile
-    from retriever.lib.repository import check_for_updates
 
-    check_for_updates()
```

Comment:
Should not download all the scripts during installation.

Reply:
Hey Harshit, I think this was what was mentioned by Ethan. So he envisioned us still downloading everything since we have a relatively small scripts size still. On first install, everything should get installed, so lets reverse this.

Reply:
ok, would do so.